### PR TITLE
PBJ: fix copy in Joy of Color (fixes #12293)

### LINKED ISSUE #12293: PBJ: Fix copy on Joy of Color
Requested copy edits (Mozilla only): https://docs.google.com/document/d/1grKBVRcWPu4mpLOAlvroyGn3IYPQg5wM80fe6yn_n3o/edit

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Requested copy edits (Mozilla only): https://docs.google.com/document/d/1grKBVRcWPu4mpLOAlvroyGn3IYPQg5wM80fe6yn_n3o/edit

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12293

## Screenshots

edits

## Testing

To test this work:
http://localhost:8000/en-US/stories/joy-of-color/

edits in 2nd and 3rd paragraph under Why the Web Is Gray

### CODE REVIEW

File: bedrock/stories/templates/stories/articles/joy-of-color.html [hunk 1]
```diff
@@ -81,9 +81,9 @@
...
+    <p>The somber tones that plague digital design have a parallel to modern architecture: design in the service of efficiency, reduced to its most simple form. The modernist movement of the early 20th century turned sharply away from superfluous decoration, including maximalist “ornamentation” (think of turning away from centuries of maximalist decoration: chinoiserie, art nouveau, the arts & crafts movement, etc.). Lee referred to “Ornament and Crime,” a manifesto in which architect Adolph Loos decried the use of decoration for decoration’s sake, in favor of useful objects as influential in the shift toward  rectilinear buildings that were reduced to the most simplified form with no “unnecessary” features. The theory was that only those who lacked restraint—children and people in non-Western societies—favored overtly decorative design fraught with unnecessary use of color (color used without purpose or intention).</p>
+    <p> “Holding back from ornamentation was a sign of spiritual strength, so if you paraded conspicuous decoration overwrought with an abundance of color, you were basically showcasing your own moral weakness,” said Lee referencing Loos’ essay.</p>
```
Comment (stevejalim) [inline bedrock/stories/templates/stories/articles/joy-of-color.html:85]: Not requested, but my brain expected a comma

```
- said Lee referencing Loos’ essay.
+ said Lee, referencing Loos’ essay.
```

Comment (stevejalim) [reply bedrock/stories/templates/stories/articles/joy-of-color.html:85]: There are some other copy edits we _could_ do but will Slack about them

**Review by stevejalim — APPROVED** [review]
r+wc

File: bedrock/stories/templates/stories/articles/joy-of-color.html (hunk 1 above)
Comment (maureenlholland) [reply bedrock/stories/templates/stories/articles/joy-of-color.html]: Adding comma, my brain was looking for the same pause

**Pull request merged**